src/ExpressionEvaluator/Core/Test/FunctionResolver/FunctionResolverTest.csproj

```diff
@@ -81,5 +81,8 @@
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
```

Comment:
I'd rather add it to the common settings file. We set ``` <RoslynProjectType>UnitTest</RoslynProjectType>``` for all projects, so we can base adding this item upon that.